packages/logger/src/Logger.ts

```diff
@@ -14,7 +14,7 @@ import { format, inspect, isNullOrUndefined } from "util";
 import { ImperativeError } from "../../error";
 import * as StackTrace from "stack-trace";
 import * as path from "path";
-import { TextUtils } from "../../utilities";
+import { TextUtils } from "../../utilities/src/TextUtils";
```

Comment:
Same as previous comment (completely fine with letting this as-is, just something I noticed during review)